benchmarks/benchmarks/analysis/distances.py

```diff
@@ -11,120 +17,243 @@
 except:
     pass
 
-class DistancesBench(object):
-    """Benchmarks for MDAnalysis.analysis.distances
-    functions.
+
+class BetweenBench(object):
+    """Benchmark for the MDAnalysis.analysis.distances.between
+    function.
     """
+    u = MDAnalysis.Universe(GRO)
+    tri_clinic_dimensions = u.dimensions
```

Comment:
What is the purpose of `tri_clinic_dimensions` here and elsewhere? 

Checking this feature branch locally--it seems to be assigned to but never used?

`git grep -E 'tri_clinic_dimensions'`

```
benchmarks/benchmarks/analysis/distances.py:    tri_clinic_dimensions = u.dimensions
benchmarks/benchmarks/analysis/distances.py:    tri_clinic_dimensions = u.dimensions
benchmarks/benchmarks/analysis/distances.py:    tri_clinic_dimensions = u.dimensions
```

Reply:
Should have been deleted, sorry, thanks for catching it.